specifications/api/swagger.yaml

```diff
@@ -3574,6 +3565,9 @@ components:
              containing helpful information.
         delegations: *transactionDelegations
         validity_interval: *ApiValidityInterval
+        hex_output:
+          type: boolean
+          default: false
```

Comment:
```suggestion
        encoding:
          type: string
          enum: ["base16", "base64"]
```

Reply:
I would agree with making this an `enum`.